watchman/checks.py

```diff
@@ -81,3 +95,7 @@ def email():
 
 def storage():
     return {"storage": _check_storage()}
+
+
+def amqp():
+    return {'amqp': _check_ampq(settings.AMQP_CONNECTIONS)}
```

Comment:
There is no standard, but we talked offline 6 months ago about using this type of convention following along with how DATABASES and CACHES are defined.